### PR TITLE
View comments

### PR DESCRIPTION
**Pull request by Scodia619**

### CODE REVIEW

File: nc-news-fe/api.js [hunk 1]
```diff
@@ -1,11 +1,31 @@
...
-    return axios.get("https://nc-news-a8e3.onrender.com/api/articles")
+const api = axios.create({
+    baseURL: "https://nc-news-a8e3.onrender.com/api"})
+
+export const getAllArticles = () => {
+    return api.get("/articles")
         .then(res => res.data.articles)
         .catch(error => {
             console.error('Error fetching data:', error);
+            throw error; // Re-throw the error to handle it where the function is called
```
Comment (popatre) [inline nc-news-fe/api.js]: Your catch should be at the end of your chain - you dont need to catch or do anything with the error in this file

**Review by popatre — CHANGES_REQUESTED** [review]
Nice work so far. 

- [ ] You're catching your error then throwing it for no reason. Do you need to catch it there, only to just throw it again?
- [ ] You should add some feedback for when an article has no comments

File: nc-news-fe/src/components/CommentContent.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            <input
+              type="text"
+              value={comment}
+              onChange={(e) => setComment(e.target.value)}
+            />
+          </label>
+          <button>Add</button>
+        </div>
+      </form>
+      <h2>Most Recent Comment</h2>
```
Comment (nc-modou) [inline nc-news-fe/src/components/CommentContent.jsx:22]: Your label at the bottom of the page reads 'most recent comment' but it doesn't actually show a comment? This is a bit confusing as the user might expect to be seeing a comment here' but it's blank

File: nc-news-fe/src/components/CommentContent.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      <button onClick={() => setShowComments(true)}>View All Comments</button>
+      {comments.length > 0 && showComments ? (
```
Comment (nc-modou) [inline nc-news-fe/src/components/CommentContent.jsx:24]: This button is a bit small and tucked away in the corner - would make for a better user experience to make it larger and perhaps centered if that works with the design of your site

File: nc-news-fe/src/components/CommentContent.jsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        <div>
+          {comments.map((comment) => (
+            <CommentCard key={comment.comment_id} comment={comment} />
+          ))}
+        </div>
+      ) : showComments ? (
+        <p>No Comments</p>
```
Comment (nc-modou) [inline nc-news-fe/src/components/CommentContent.jsx:31]: This message could be  bit more noticeable styling wise

**Review by nc-modou — COMMENTED** [review]
Nice job on the comments section, looks pretty good so far.

Comments part of the page works well overall, but left some line by line comments to address to polish it off

**Pull request merged**